pages/common/gnmic-get.md

```diff
@@ -0,0 +1,20 @@
+# gnmic get
+
+> Get a snapshot of a gnmi network device operation data.
+> More information: <https://gnmic.kmrd.dev/cmd/get>.
+
+- Get a snapshot of the device state at {{path}}:
```

Comment:
```suggestion
- Get a snapshot of the device state at specific path:
```

It is not mentioned in our style guide what to do with placeholders inside descriptions (but it should be). Anyway I think we should not use them anywhere outside code examples.